pkg/sanity/controller.go

```diff
@@ -44,6 +45,22 @@ func TestVolumeSize(sc *TestContext) int64 {
 	return sc.Config.TestVolumeSize
 }
 
+func TestVolumeCapabilityWithAccessType(sc *TestContext, m csi.VolumeCapability_AccessMode_Mode) *csi.VolumeCapability {
```

Comment:
nit: `Test` prefix implies that it actually tests something.

Reply:
I was just following the way how getters are named in that file, e. g.  `TestVolumeSize`, `TestVolumeExpandSize`

Reply:
Oh, OK.